addons/survey/wizard/survey_invite.py

```diff
@@ -60,6 +60,8 @@ def _get_default_author(self):
     deadline = fields.Datetime(string="Answer deadline")
     send_email = fields.Boolean(compute="_compute_send_email",
                                 inverse="_inverse_send_email")
+    comment = fields.Html(string="Comments")
```

Comment:
the field should be text instead of Html as we are appending extra comments to
existing HTML body. (apply to both place)